strawberryfields/_version.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 """Version information.
-   Version number (major.minor.patch[-label])
+Version number (major.minor.patch[-label])
 """
 
 __version__ = "0.24.0-dev"
```

Comment:
I guess this should be bumped (either here or in a subsequent PR).

Reply:
don't the SF people use a dev version until the next release is produced? Not sure we want to make a new release yet. See Changelog - there are many commits per release.

Reply:
Ah, ok, we can make these changes without creating a release. :+1: